### PR TITLE
Feat : added basic routing and folder structure

### PR DESCRIPTION
**Pull request by raman1999**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,6 +7,10 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
+  <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/stylesheets/components.css">
+  <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/landing.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
```
Comment (rdshinde) [inline public/index.html:12]: I think CSS stylesheets' imports are preferred in CSS files like App.css
Why are you linking them in index, any specific reason?

**Review by rdshinde — APPROVED** [review]
Just a single comment.
Otherwise everything looks fine.

File: public/index.html (hunk 1 above)
Comment (raman1999) [reply public/index.html:12]: I have added it here to use preconnect with it  for faster loading, as css import is slow

**Review by aishwarya-bm — APPROVED** [review]
Everything looks good, no suggestion.
The naming of components and their breakup looks good with clean code

**Pull request merged**